src/main/java/emissary/output/DropOffUtil.java

```diff
@@ -287,42 +287,6 @@ public String getOutputDirectory() {
         return this.placeOutputData;
     }
 
-    public String getOutputDirectory(final IBaseDataObject DO) {
```

Comment:
Verified that there were no references to this overload in downstream repositories.